# Chinese (Taiwan) translation for apport
# Copyright (c) (c) 2006 Canonical Ltd, and Rosetta Contributors 2006
# This file is distributed under the same license as the apport package.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2006.
#
msgid ""
msgstr ""
"Project-Id-Version: apport\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2023-02-10 12:34+0100\n"
"PO-Revision-Date: 2019-08-13 14:40+0000\n"
"Last-Translator: Cheng-Chia Tseng <pswo10680@gmail.com>\n"
"Language-Team: Chinese (Taiwan) <zh_TW@li.org>\n"
"Language: \n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"X-Launchpad-Export-Date: 2022-12-07 14:36+0000\n"
"X-Generator: Launchpad (build 41e7553f8097ce3a683f90e835708cbe7bf0006c)\n"

#: ../apport/com.ubuntu.apport.policy.in.h:1
msgid "Collect system information"
msgstr "蒐集系統資訊"

#: ../apport/com.ubuntu.apport.policy.in.h:2
msgid ""
"Authentication is required to collect system information for this problem "
"report"
msgstr "若要為此問題回報蒐集系統資訊需先通過身份驗證"

#: ../apport/com.ubuntu.apport.policy.in.h:3
msgid "System problem reports"
msgstr "系統問題報告"

#: ../apport/com.ubuntu.apport.policy.in.h:4
msgid "Please enter your password to access problem reports of system programs"
msgstr "請輸入密碼以存取系統程式的問題報告"

#: ../apport/ui.py:166
msgid "This package does not seem to be installed correctly"
msgstr "這個軟體包似乎尚未正確安裝"

#: ../apport/ui.py:176
#, python-format
msgid ""
"This does not seem to be an official %s package. Please retry after updating "
"the indexes of available packages, if that does not work then remove related "
"third party packages and try again."
msgstr ""

#: ../apport/ui.py:206
#, python-format
msgid ""
"You have some obsolete package versions installed. Please upgrade the "
"following packages and check if the problem still occurs:\n"
"\n"
"%s"
msgstr ""
"您安裝了某些已過時的軟體包。請升級以下軟體包以及檢查問題是否繼續產生：\n"
"\n"
"%s"

#: ../apport/ui.py:369
msgid "unknown program"
msgstr "未知程式"

#: ../apport/ui.py:372
#, python-format
msgid "Sorry, the program \"%s\" closed unexpectedly"
msgstr "對不起，\"%s\" 程式已不正常關閉"

#: ../apport/ui.py:375 ../apport/ui.py:1907
#, python-format
msgid "Problem in %s"
msgstr "在 %s 中的問題"

#: ../apport/ui.py:380
msgid ""
"Your computer does not have enough free memory to automatically analyze the "
"problem and send a report to the developers."
msgstr "閣下電腦無足夠可用記憶體供自動分析問題並向開發者傳送報告。"

#: ../apport/ui.py:440 ../apport/ui.py:458 ../apport/ui.py:631
#: ../apport/ui.py:640 ../apport/ui.py:877 ../apport/ui.py:1685
#: ../apport/ui.py:1868 ../apport/ui.py:1873
msgid "Invalid problem report"
msgstr "無效的問題回報"

#: ../apport/ui.py:441
msgid "You are not allowed to access this problem report."
msgstr "您無權存取此問題報告。"

#: ../apport/ui.py:449
msgid "Error"
msgstr "錯誤"

#: ../apport/ui.py:451
msgid "There is not enough disk space available to process this report."
msgstr "無足夠磁碟空間處理此報告。"

#: ../apport/ui.py:489
msgid "No PID specified"
msgstr "沒有指定 PID"

#: ../apport/ui.py:491
msgid "You need to specify a PID. See --help for more information."
msgstr "您需要指定一個 PID。更多說明請見 --help。"

#: ../apport/ui.py:502 ../apport/ui.py:607
msgid "Invalid PID"
msgstr "PID 無效"

#: ../apport/ui.py:503
msgid "The specified process ID does not exist."
msgstr "指定的程序 ID 不存在。"

#: ../apport/ui.py:508
msgid "Not your PID"
msgstr "不是您的 PID"

#: ../apport/ui.py:509
msgid "The specified process ID does not belong to you."
msgstr "指定的程序 ID 不屬於您。"

#: ../apport/ui.py:564
msgid "No package specified"
msgstr "未指定軟體包"

#: ../apport/ui.py:566
msgid ""
"You need to specify a package or a PID. See --help for more information."
msgstr "您要指定軟體包或 PID。更多資訊請見 --help。"

#: ../apport/ui.py:594
msgid "Permission denied"
msgstr "權限不足"

#: ../apport/ui.py:596
msgid ""
"The specified process does not belong to you. Please run this program as the "
"process owner or as root."
msgstr "指定的程序並不屬於您。請以程序擁有者或是 root 身份執行此程式。"

#: ../apport/ui.py:609
msgid "The specified process ID does not belong to a program."
msgstr "指定的程序 ID 並不屬於程式。"

#: ../apport/ui.py:633
#, python-format
msgid "Symptom script %s did not determine an affected package"
msgstr "徵狀判斷指令稿 %s 無法判斷出受影響的軟體包"

#: ../apport/ui.py:641
#, python-format
msgid "Package %s does not exist"
msgstr "%s 軟體包不存在"

#: ../apport/ui.py:670 ../apport/ui.py:882 ../apport/ui.py:915
#: ../apport/ui.py:925
msgid "Cannot create report"
msgstr "無法建立報告"

#: ../apport/ui.py:685 ../apport/ui.py:744 ../apport/ui.py:762
msgid "Updating problem report"
msgstr "正在更新問題報告"

#: ../apport/ui.py:687
msgid ""
"You are not the reporter or subscriber of this problem report, or the report "
"is a duplicate or already closed.\n"
"\n"
"Please create a new report using \"apport-bug\"."
msgstr ""
"您不是此問題回報的提報者或訂閱者，或者這個報告為複本或已經關閉。\n"
"\n"
"請使用「apport-bug」來建立一份新的問題回報。"

#: ../apport/ui.py:699
msgid ""
"You are not the reporter of this problem report. It is much easier to mark a "
"bug as a duplicate of another than to move your comments and attachments to "
"a new bug.\n"
"\n"
"Subsequently, we recommend that you file a new bug report using \"apport-"
"bug\" and make a comment in this bug about the one you file.\n"
"\n"
"Do you really want to proceed?"
msgstr ""
"您不是此問題回報的提報者。比起將您的備註與附件移至新的臭蟲 (軟體錯誤) 錯誤報"
"告，將臭蟲標記為另一個問題回報的複本更容易許多。\n"
"\n"
"不過，我們建議您使用「apport-bug」來提交回報新的臭蟲回報，並請寫下與您提交的"
"臭蟲回報相關的備註。\n"
"\n"
"您真的想要繼續嗎？"

#: ../apport/ui.py:745 ../apport/ui.py:763
msgid "No additional information collected."
msgstr "沒有收集到額外的資訊。"

#: ../apport/ui.py:826
msgid "What kind of problem do you want to report?"
msgstr "您想要回報何種問題？"

#: ../apport/ui.py:847
msgid "Unknown symptom"
msgstr "不知名徵狀"

#: ../apport/ui.py:848
#, python-format
msgid "The symptom \"%s\" is not known."
msgstr "「%s」徵狀不為人所知。"

#: ../apport/ui.py:884
msgid ""
"The window option cannot be used on Wayland.\n"
"\n"
"Please find the window's process ID and then run 'ubuntu-bug <process ID>'.\n"
"\n"
"The process ID can be found by running the System Monitor application. In "
"the Processes tab, scroll until you find the correct application. The "
"process ID is the number listed in the ID column."
msgstr ""

#: ../apport/ui.py:900
msgid ""
"After closing this message please click on an application window to report a "
"problem about it."
msgstr "在關閉本訊息後，請點擊應用程式視窗來回報有關它的訊息。"

#: ../apport/ui.py:917 ../apport/ui.py:926
msgid "xprop failed to determine process ID of the window"
msgstr "xprop 無法得知該視窗的程序 ID"

#: ../apport/ui.py:941
#, python-format
msgid "%(prog)s <report number>"
msgstr ""

#: ../apport/ui.py:942
msgid "Specify package name."
msgstr "指定軟體包名稱。"

#: ../apport/ui.py:949 ../apport/ui.py:1082
msgid "Add an extra tag to the report. Can be specified multiple times."
msgstr "加入額外標籤至回報中。可以多次分別指定。"

#: ../apport/ui.py:987
#, python-format
msgid ""
"%(prog)s [options] [symptom|pid|package|program path|.apport/.crash file]"
msgstr ""

#: ../apport/ui.py:998
msgid ""
"Start in bug filing mode. Requires --package and an optional --pid, or just "
"a --pid. If neither is given, display a list of known symptoms. (Implied if "
"a single argument is given.)"
msgstr ""
"啟動為臭蟲歸檔模式。需要 --package 或再加上 --pid，或是只需 --pid。若兩者都未"
"給予，顯示已知徵狀列表。(暗示若只有給予單一參數。)"

#: ../apport/ui.py:1007
msgid "Click a window as a target for filing a problem report."
msgstr "請點擊要作為問題報告目標發送的視窗。"

#: ../apport/ui.py:1016
msgid "Start in bug updating mode. Can take an optional --package."
msgstr "以臭蟲更新模式啟動。可以加上選用的參數 --package。"

#: ../apport/ui.py:1025
msgid ""
"File a bug report about a symptom. (Implied if symptom name is given as only "
"argument.)"
msgstr "建立關於症狀的錯誤報告。(暗示是否只要提供症狀名稱作為引數。)"

#: ../apport/ui.py:1034
msgid ""
"Specify package name in --file-bug mode. This is optional if a --pid is "
"specified. (Implied if package name is given as only argument.)"
msgstr ""
"在 --file-bug 模式中指定軟體包名稱。若指定 --pid 則前者不加亦可。(表示是否只"
"要提供 pid 為引數。)"

#: ../apport/ui.py:1045
msgid ""
"Specify a running program in --file-bug mode. If this is specified, the bug "
"report will contain more information.  (Implied if pid is given as only "
"argument.)"
msgstr ""
"在 --file-bug 模式中指定一個正在執行中的程式。如果有指定，該臭蟲回報將包含更"
"多資訊。(表示是否只要提供 pid 為引數。)"

#: ../apport/ui.py:1053
msgid "The provided pid is a hanging application."
msgstr "提供的 pid 是懸凍的應用程式。"

#: ../apport/ui.py:1061
#, python-format
msgid ""
"Report the crash from given .apport or .crash file instead of the pending "
"ones in %s. (Implied if file is given as only argument.)"
msgstr ""
"從特定的 .apport 或 .crash 檔案來回報當掉情形，代替等待中的 %s。（暗示若特定"
"檔案是唯一給予的參數。）"

#: ../apport/ui.py:1071
msgid ""
"In bug filing mode, save the collected information into a file instead of "
"reporting it. This file can then be reported later on from a different "
"machine."
msgstr ""
"在臭蟲歸檔模式中，收集的資訊將儲存在檔案中而不是將它回報。這個檔案可以稍後從"
"不同機器回報。"

#: ../apport/ui.py:1090
msgid "Print the Apport version number."
msgstr "列印 Apport 版本號碼。"

#: ../apport/ui.py:1252
msgid ""
"This will launch apport-retrace in a terminal window to examine the crash."
msgstr "這會在終端機視窗中啟動 apport-retrace 以檢查程式當掉原因。"

#: ../apport/ui.py:1256
msgid "Run gdb session"
msgstr "執行 gdb 作業階段"

#: ../apport/ui.py:1257
msgid "Run gdb session without downloading debug symbols"
msgstr "執行 gdb 作業階段而不下載除錯符號"

#. TRANSLATORS: %s contains the crash report file name
#: ../apport/ui.py:1259
#, python-format
msgid "Update %s with fully symbolic stack trace"
msgstr "以完整的符號堆疊追蹤來更新 %s"

#: ../apport/ui.py:1321
msgid "Can't remember send report status settings"
msgstr "無法儲存「傳送回報狀態」設定"

#: ../apport/ui.py:1325
msgid ""
"Saving crash reporting state failed. Can't set auto or never reporting mode."
msgstr "儲存回報當機狀態時發生錯誤，無法設定自動或永不回報模式。"

#: ../apport/ui.py:1408 ../apport/ui.py:1421
msgid ""
"This problem report applies to a program which is not installed any more."
msgstr "此問題報告用於不再安裝的程式。"

#: ../apport/ui.py:1445
#, python-format
msgid ""
"The problem happened with the program %s which changed since the crash "
"occurred."
msgstr "發生問題的程式 %s 自上次遭遇當掉之後已有變動。"

#: ../apport/ui.py:1506 ../apport/ui.py:1626 ../apport/ui.py:1877
msgid "This problem report is damaged and cannot be processed."
msgstr "此問題報告已損壞故無法處理。"

#: ../apport/ui.py:1515
msgid "This report is about a package that is not installed."
msgstr "這個是關於未安裝軟體包的回報。"

#: ../apport/ui.py:1523
msgid "An error occurred while attempting to process this problem report:"
msgstr "試圖處理問題回報時遭遇錯誤："

#: ../apport/ui.py:1540
msgid ""
"You have two versions of this application installed, which one do you want "
"to report a bug against?"
msgstr "此應用程式您安裝了兩個版本，您想對哪一個版本回報臭蟲？"

#: ../apport/ui.py:1545
#, python-format
msgid "%s snap"
msgstr "%s snap"

#: ../apport/ui.py:1546
#, python-format
msgid "%s deb package"
msgstr "%s deb 套件包"

#: ../apport/ui.py:1583
#, python-format
msgid "%s is provided by a snap published by %s. Contact them via %s for help."
msgstr ""

#: ../apport/ui.py:1588
#, python-format
msgid ""
"%s is provided by a snap published by %s. No contact address has been "
"provided; visit the forum at https://forum.snapcraft.io/ for help."
msgstr ""
"%s 是由 %s 所發布的 snap 。開發者沒有提供聯絡地址；請前往論壇 https://forum."
"snapcraft.io/ 尋求協助。"

#: ../apport/ui.py:1687
msgid "Could not determine the package or source package name."
msgstr "無法決定軟體包或原始碼軟體包名稱。"

#: ../apport/ui.py:1712
msgid "Unable to start web browser"
msgstr "無法啟動網頁瀏覽器"

#: ../apport/ui.py:1713
#, python-format
msgid "Unable to start web browser to open %s."
msgstr "無法啟動網頁瀏覽器以開啟 %s。"

#: ../apport/ui.py:1823
msgid "Network problem"
msgstr "網路問題"

#: ../apport/ui.py:1827
msgid ""
"Cannot connect to crash database, please check your Internet connection."
msgstr "無法連接到當機資料庫，請檢查您的網路連線。"

#: ../apport/ui.py:1859
msgid "Memory exhaustion"
msgstr "記憶體耗盡"

#: ../apport/ui.py:1861
msgid "Your system does not have enough memory to process this crash report."
msgstr "您的系統沒有足夠的記憶體來處理此份當機報告。"

#: ../apport/ui.py:1912
#, python-format
msgid ""
"The problem cannot be reported:\n"
"\n"
"%s"
msgstr ""
"無法報告此問題：\n"
"\n"
"%s"

#: ../apport/ui.py:1968 ../apport/ui.py:1980
msgid "Problem already known"
msgstr "已知問題"

#: ../apport/ui.py:1970
msgid ""
"This problem was already reported in the bug report displayed in the web "
"browser. Please check if you can add any further information that might be "
"helpful for the developers."
msgstr ""
"此問題在網路瀏覽器顯示之臭蟲報告中已回報。請檢查您是否能新增任何可能對開發者"
"有幫助的進一步資訊。"

#: ../apport/ui.py:1982
msgid "This problem was already reported to developers. Thank you!"
msgstr "此問題已經回報給開發者。感謝您！"

#: ../bin/apport-cli.py:84
msgid "Press any key to continue..."
msgstr "按任意鍵繼續..."

#: ../bin/apport-cli.py:91
msgid "What would you like to do? Your options are:"
msgstr "您要做什麼？選擇有："

#: ../bin/apport-cli.py:104
#, python-format
msgid "Please choose (%s):"
msgstr "請選擇 (%s):"

#: ../bin/apport-cli.py:172
#, python-format
msgid "(%i bytes)"
msgstr "(%i 位元組)"

#: ../bin/apport-cli.py:174 ../gtk/apport-gtk.py:169 ../kde/apport-kde.py:405
msgid "(binary data)"
msgstr "（二進制資料）"

#: ../bin/apport-cli.py:209 ../gtk/apport-gtk.py:208 ../gtk/apport-gtk.ui.h:6
#: ../kde/apport-kde.py:198
msgid "Send problem report to the developers?"
msgstr "是否傳送問題報告予開發者？"

#: ../bin/apport-cli.py:211
msgid ""
"After the problem report has been sent, please fill out the form in the\n"
"automatically opened web browser."
msgstr ""
"在問題報告傳送後，請填寫自動開啟之\n"
"網路瀏覽器中的表格。"

#: ../bin/apport-cli.py:218
#, python-format
msgid "&Send report (%s)"
msgstr "傳送報告 (%s)(&S)"

#: ../bin/apport-cli.py:223
msgid "&Examine locally"
msgstr "於本地端檢查(&E)"

#: ../bin/apport-cli.py:227
msgid "&View report"
msgstr "檢視報告(&V)"

#: ../bin/apport-cli.py:230
msgid "&Keep report file for sending later or copying to somewhere else"
msgstr "保留回報檔，稍後傳送或複製至其他位置(&K)"

#: ../bin/apport-cli.py:235
msgid "Cancel and &ignore future crashes of this program version"
msgstr "取消及忽略未來此程式版本的當機(&I)"

#: ../bin/apport-cli.py:238 ../bin/apport-cli.py:326 ../bin/apport-cli.py:360
#: ../bin/apport-cli.py:383
msgid "&Cancel"
msgstr "取消(&C)"

#: ../bin/apport-cli.py:267
msgid "Problem report file:"
msgstr "問題回報檔："

#: ../bin/apport-cli.py:273 ../bin/apport-cli.py:278
msgid "&Confirm"
msgstr "確認(&C)"

#: ../bin/apport-cli.py:277
#, python-format
msgid "Error: %s"
msgstr "錯誤：%s"

#: ../bin/apport-cli.py:283 ../kde/apport-kde.py:455
msgid "Collecting problem information"
msgstr "正在收集問題資訊"

#: ../bin/apport-cli.py:285
msgid ""
"The collected information can be sent to the developers to improve the\n"
"application. This might take a few minutes."
msgstr ""
"收集的資訊可傳送予開發者以改進此\n"
"應用程式。這可能需要數分鐘。"

#: ../bin/apport-cli.py:301 ../gtk/apport-gtk.ui.h:17 ../kde/apport-kde.py:504
msgid "Uploading problem information"
msgstr "正在上傳問題資訊"

#: ../bin/apport-cli.py:303
msgid ""
"The collected information is being sent to the bug tracking system.\n"
"This might take a few minutes."
msgstr ""
"已收集的資訊將傳送至臭蟲追蹤系統。\n"
"這可能需要數分鐘。"

#: ../bin/apport-cli.py:359
msgid "&Done"
msgstr "完成(&D)"

#: ../bin/apport-cli.py:365
msgid "none"
msgstr "無"

#: ../bin/apport-cli.py:367
#, python-format
msgid "Selected: %s. Multiple choices:"
msgstr "已選擇：%s。多種選擇："

#: ../bin/apport-cli.py:384
msgid "Choices:"
msgstr "選項："

#: ../bin/apport-cli.py:398
msgid "Path to file (Enter to cancel):"
msgstr "檔案路徑 (按 Enter 取消)："

#: ../bin/apport-cli.py:404
msgid "File does not exist."
msgstr "檔案不存在。"

#: ../bin/apport-cli.py:406
msgid "This is a directory."
msgstr "這是個目錄。"

#: ../bin/apport-cli.py:412
msgid "To continue, you must visit the following URL:"
msgstr "欲繼續，您必須造訪以下網址："

#: ../bin/apport-cli.py:415
msgid ""
"You can launch a browser now, or copy this URL into a browser on another "
"computer."
msgstr "您現在可以啟動瀏覽器，或複製此網址到另一臺電腦的瀏覽器中。"

#: ../bin/apport-cli.py:421
msgid "Launch a browser now"
msgstr "現在啟動瀏覽器"

#: ../bin/apport-cli.py:437
msgid "No pending crash reports. Try --help for more information."
msgstr "無待處理之當機報告。更多資訊請試試 --help。"

#: ../bin/apport-retrace.py:44
msgid "Do not put the new traces into the report, but write them to stdout."
msgstr "不要將新的錯誤追溯放在報告中，但輸出至標準輸出（stdout）。"

#: ../bin/apport-retrace.py:53
msgid ""
"Start an interactive gdb session with the report's core dump (-o ignored; "
"does not rewrite report)"
msgstr "啟動新的互動式 gdb 連線階段及核心傾印報告（-o 忽略：不會複寫報告）"

#: ../bin/apport-retrace.py:62
msgid ""
"Write modified report to given file instead of changing the original report"
msgstr "寫入已修改的報告至特定的檔案來替代改變原始報告"

#: ../bin/apport-retrace.py:72
msgid "Remove the core dump from the report after stack trace regeneration"
msgstr "在堆疊追溯重新產生後從報告中移除核心傾印"

#: ../bin/apport-retrace.py:80
msgid "Override report's CoreFile"
msgstr "覆蓋報告的核心檔案（CoreFile）"

#: ../bin/apport-retrace.py:86
msgid "Override report's ExecutablePath"
msgstr "覆蓋報告的可執行路徑（ExecutablePath）"

#: ../bin/apport-retrace.py:92
msgid "Override report's ProcMaps"
msgstr "覆蓋報告的行程對映（ProcMaps）"

#: ../bin/apport-retrace.py:98
msgid "Rebuild report's Package information"
msgstr "重建報告的軟體包資訊"

#: ../bin/apport-retrace.py:105
msgid ""
"Build a temporary sandbox and download/install the necessary packages and "
"debug symbols in there; without this option it assumes that the necessary "
"packages and debug symbols are already installed in the system. The argument "
"points to the packaging system configuration base directory; if you specify "
"\"system\", it will use the system configuration files, but will then only "
"be able to retrace crashes that happened on the currently running release."
msgstr ""
"建立暫時沙箱，並下載/安裝需要的軟體包與除錯符於該沙箱中；若不採用此選項，將假"
"定需要的軟體包與除錯符都已經安裝在系統之內。引數指向打包系統組態的基礎目錄；"
"如果您有指定「system」(系統) 的話，它會使用系統組態檔，但是接著只能回追發生於"
"目前執行中發行版的當掉情形。"

#: ../bin/apport-retrace.py:119
msgid ""
"Build another temporary sandbox for installing gdb and its dependencies "
"using the same release as the report rather than whatever version of gdb you "
"have installed."
msgstr ""
"建置另一個暫時沙盒以安裝 gdb 和其相關依賴以使用相同發行版作回報，而不是採用您"
"本機已安裝的 gdb 版本。"

#: ../bin/apport-retrace.py:129
msgid "Report download/install progress when installing packages into sandbox"
msgstr "當軟體包安裝至沙箱時回報下載/安裝進度"

#: ../bin/apport-retrace.py:136
msgid "Prepend timestamps to log messages, for batch operation"
msgstr "在記錄訊息前加上時間戳記，供批次操作"

#: ../bin/apport-retrace.py:142
msgid ""
"Create and use third-party repositories from origins specified in reports"
msgstr "製作並使用指定的第三方軟體庫來源置於報告中"

#: ../bin/apport-retrace.py:150
msgid "Cache directory for packages downloaded in the sandbox"
msgstr "為下載於沙箱內的軟體包製作目錄快取"

#: ../bin/apport-retrace.py:156
msgid ""
"Directory for unpacked packages. Future runs will assume that any already "
"downloaded package is also extracted to this sandbox."
msgstr ""
"解開軟體包的目錄。未來執行時將假定任何已下載的軟體包也都抽出於此沙盒中。"

#: ../bin/apport-retrace.py:167 ../bin/apport-valgrind.py:97
msgid ""
"Install an extra package into the sandbox (can be specified multiple times)"
msgstr "安裝額外軟體包至沙箱內 (可以指定多次)"

#: ../bin/apport-retrace.py:174
msgid ""
"Path to a file with the crash database authentication information. This is "
"used when specifying a crash ID to upload the retraced stack traces (only if "
"neither -g, -o, nor -s are specified)"
msgstr ""
"有當機資料庫認證資訊的檔案路徑。在指定某個當機 ID 來上傳重新追蹤的堆疊追蹤時"
"可使用 (只有在 -g、-o 或 -s 都沒有被指定)"

#: ../bin/apport-retrace.py:184
msgid ""
"Display retraced stack traces and ask for confirmation before sending them "
"to the crash database."
msgstr "顯示重新追蹤的堆疊追蹤，並在送出它們到當機資料庫之前先請求確認。"

#: ../bin/apport-retrace.py:192
msgid "Path to the duplicate sqlite database (default: no duplicate checking)"
msgstr "重複的 sqlite 資料庫路徑（預設：無重複檢查）"

#: ../bin/apport-retrace.py:200
msgid "Do not add StacktraceSource to the report."
msgstr "不要在回報中增加 StacktraceSource"

#: ../bin/apport-retrace.py:213
msgid "You cannot use -C without -S. Stopping."
msgstr "不能在沒有 -S 的情況下使用 -C。結束處理。"

#. translators: don't translate y/n,
#. apport currently only checks for "y"
#: ../bin/apport-retrace.py:243
msgid "OK to send these as attachments? [y/n]"
msgstr "允許傳送這些附件？"

#: ../bin/apport-unpack.py:29
#, python-format
msgid "%(prog)s <report> <target directory>"
msgstr ""

#: ../bin/apport-unpack.py:31
msgid "Report file to unpack"
msgstr ""

#: ../bin/apport-unpack.py:33
msgid "directory to unpack report to"
msgstr ""

#: ../bin/apport-unpack.py:59
msgid "Destination directory exists and is not empty."
msgstr "目的目錄存在且不是空的。"

#: ../bin/apport-valgrind.py:37
msgid "See man page for details."
msgstr "詳情見 man page。"

#: ../bin/apport-valgrind.py:46
msgid "specify the log file name produced by valgrind"
msgstr "提供由 valgrind 產生的日誌檔名"

#: ../bin/apport-valgrind.py:52
msgid ""
"reuse a previously created sandbox dir (SDIR) or, if it does not exist, "
"create it"
msgstr "重新使用之前建立的沙箱資料夾 (SDIR) 若不存在則新增一個"

#: ../bin/apport-valgrind.py:60
msgid ""
"do  not  create  or reuse a sandbox directory for additional debug symbols "
"but rely only on installed debug symbols."
msgstr ""
"不能新增或使用之前的沙盒給額外的Debug symbol 用途,但僅依靠安裝的 debug "
"symbols"

#: ../bin/apport-valgrind.py:69
msgid ""
"reuse a previously created cache dir (CDIR) or, if it does not exist, create "
"it"
msgstr "重新使用之前建立的暫存資料夾 (CDIR) 若不存在則新增一個"

#: ../bin/apport-valgrind.py:78
msgid "report download/install progress when installing packages into sandbox"
msgstr "當軟體包安裝至沙箱時回報下載/安裝進度"

#: ../bin/apport-valgrind.py:86
msgid ""
"the executable that is run under valgrind's memcheck tool for memory leak "
"detection"
msgstr ""

#: ../bin/apport-valgrind.py:129
#, python-format
msgid "Error: %s is not an executable. Stopping."
msgstr "錯誤：%s 並非可執行檔。結束處理。"

#: ../data/apportcheckresume.py:68
msgid ""
"This occurred during a previous suspend, and prevented the system from "
"resuming properly."
msgstr "在上次暫停時遭遇這個問題，並使系統無法正常恢復。"

#: ../data/apportcheckresume.py:73
msgid ""
"This occurred during a previous hibernation, and prevented the system from "
"resuming properly."
msgstr "在上次休眠時遭遇這個問題，並使系統無法正常恢復。"

#: ../data/apportcheckresume.py:81
msgid ""
"The resume processing hung very near the end and will have appeared to have "
"completed normally."
msgstr "回復行程在非常接近結束的時候停止回應，且將會表現出正常完成的情形。"

#: ../data/kernel_oops.py:30
msgid "Your system might become unstable now and might need to be restarted."
msgstr "您的系統現在也許會不穩定且需要重新啟動。"

#: ../gtk/apport-gtk.desktop.in.h:1 ../kde/apport-kde-mime.desktop.in.h:1
#: ../kde/apport-kde.desktop.in.h:1
msgid "Report a problem..."
msgstr "報告問題..."

#: ../gtk/apport-gtk.desktop.in.h:2 ../kde/apport-kde-mime.desktop.in.h:2
#: ../kde/apport-kde.desktop.in.h:2
msgid "Report a malfunction to the developers"
msgstr "報告問題予開發者"

#: ../gtk/apport-gtk.py:185
#, python-format
msgid "Sorry, the application %s has stopped unexpectedly."
msgstr "很抱歉，應用程式 %s 已無預期停止。"

#: ../gtk/apport-gtk.py:188
#, python-format
msgid "Sorry, %s has closed unexpectedly."
msgstr "非常抱歉，%s 已無預警關閉。"

#: ../gtk/apport-gtk.py:193 ../kde/apport-kde.py:213 ../kde/apport-kde.py:265
#, python-format
msgid "Sorry, %s has experienced an internal error."
msgstr "非常抱歉， %s 遭遇內部錯誤。"

#: ../gtk/apport-gtk.py:216 ../gtk/apport-gtk.ui.h:14 ../kde/apport-kde.py:206
msgid "Send"
msgstr "傳送"

#: ../gtk/apport-gtk.py:234 ../gtk/apport-gtk.py:660 ../gtk/apport-gtk.ui.h:11
#: ../kde/apport-kde.py:329
msgid "Show Details"
msgstr "顯示詳細資料"

#: ../gtk/apport-gtk.py:247 ../kde/apport-kde.py:257 ../kde/apport-kde.py:275
msgid "Continue"
msgstr "繼續"

#: ../gtk/apport-gtk.py:271
#, python-format
msgid "The application %s has stopped responding."
msgstr "應用程式 %s 已停止回應。"

#: ../gtk/apport-gtk.py:275
#, python-format
msgid "The program \"%s\" has stopped responding."
msgstr "程式「%s」已停止回應。"

#: ../gtk/apport-gtk.py:292 ../kde/apport-kde.py:221
#, python-format
msgid "Package: %s"
msgstr "軟體包：%s"

#: ../gtk/apport-gtk.py:298 ../kde/apport-kde.py:228
msgid "Sorry, a problem occurred while installing software."
msgstr "非常抱歉，於安裝軟體時遭遇問題。"

#: ../gtk/apport-gtk.py:310 ../gtk/apport-gtk.py:334 ../kde/apport-kde.py:237
#, python-format
msgid "The application %s has experienced an internal error."
msgstr "應用程式 %s 已遭遇某內部錯誤。"

#: ../gtk/apport-gtk.py:316 ../kde/apport-kde.py:244
#, python-format
msgid "The application %s has closed unexpectedly."
msgstr "應用程式 %s 已無預警關閉。"

#: ../gtk/apport-gtk.py:347 ../gtk/apport-gtk.ui.h:7 ../kde/apport-kde.py:270
msgid "If you notice further problems, try restarting the computer."
msgstr "若您之後仍發現問題，請嘗試重新啟動電腦。"

#: ../gtk/apport-gtk.py:352 ../kde/apport-kde.py:277
msgid "Ignore future problems of this type"
msgstr "忽略未來發生的此類型問題"

#: ../gtk/apport-gtk.py:664 ../kde/apport-kde.py:326
msgid "Hide Details"
msgstr "隱藏詳細資料"

#: ../gtk/apport-gtk.ui.h:1 ../kde/apport-kde.py:545 ../kde/apport-kde.py:588
msgid "Apport"
msgstr "Apport"

#: ../gtk/apport-gtk.ui.h:2
msgid "Cancel"
msgstr "取消"

#: ../gtk/apport-gtk.ui.h:3
msgid "OK"
msgstr "確定"

#: ../gtk/apport-gtk.ui.h:4
msgid "Crash report"
msgstr "當機回報"

#: ../gtk/apport-gtk.ui.h:5
msgid "<big><b>Sorry, an internal error happened.</b></big>"
msgstr "<big><b>非常抱歉，發生內部錯誤。</b></big>"

#: ../gtk/apport-gtk.ui.h:8
msgid "Remember this in future"
msgstr "記住此項目"

#: ../gtk/apport-gtk.ui.h:9
msgid "Ignore future problems of this program version"
msgstr "忽略本程式版本未來發生的錯誤"

#: ../gtk/apport-gtk.ui.h:10
msgid "Relaunch this application"
msgstr "重啟應用程式"

#: ../gtk/apport-gtk.ui.h:12
msgid "_Examine locally"
msgstr "於本地端檢查(_E)"

#: ../gtk/apport-gtk.ui.h:13
msgid "Don't send"
msgstr "請勿傳送"

#: ../gtk/apport-gtk.ui.h:15
msgid "<big><b>Collecting problem information</b></big>"
msgstr "<big><b>正在收集問題資訊</b></big>"

#: ../gtk/apport-gtk.ui.h:16
msgid ""
"Information is being collected that may help the developers fix the problem "
"you report."
msgstr "正在收集資訊以幫助開發者修正您報告的問題。"

#: ../gtk/apport-gtk.ui.h:18
msgid "<big><b>Uploading problem information</b></big>"
msgstr "<big><b>正在上傳問題資訊</b></big>"

#: ../gtk/apport-gtk.ui.h:19 ../kde/apport-kde.py:506
msgid ""
"The collected information is being sent to the bug tracking system. This "
"might take a few minutes."
msgstr "正將收集到的資訊傳送至臭蟲追蹤系統。這可能需要數分鐘。"

#: ../kde/apport-kde-mimelnk.desktop.in.h:1
msgid "Apport crash file"
msgstr "Apport 當機檔案"

#: ../kde/apport-kde.py:260
msgid "Leave Closed"
msgstr "維持關閉"

#: ../kde/apport-kde.py:261 ../kde/apport-kde.py:425
msgid "Relaunch"
msgstr "重新啟動"

#: ../kde/apport-kde.py:354
msgid "Username:"
msgstr "使用者名稱："

#: ../kde/apport-kde.py:355
msgid "Password:"
msgstr "密碼："

#: ../kde/apport-kde.py:454
msgid "Collecting Problem Information"
msgstr "正收集問題資訊"

#: ../kde/apport-kde.py:457
msgid ""
"The collected information can be sent to the developers to improve the "
"application. This might take a few minutes."
msgstr "收集之資訊會傳送予開發者以改善該應用程式。此舉或需數分鐘。"

#: ../kde/apport-kde.py:503
msgid "Uploading Problem Information"
msgstr "正上傳問題資訊"
